### ISSUE TITLE
[Bug]:  Wrong version (2.4.1) being returned by `opencommit --version` after updating to 2.4.2

### ISSUE THREAD

**Issue by timothy-bryant-simplisafe** [di-sukharev/opencommit#235]
### Opencommit Version

2.4.2

### Node Version

18.17.0

### NPM Version

9.6.7

### What OS are you seeing the problem on?

macOS 13.4.1 

### What happened?

After updating to 2.4.2 via `npm i -g opencommit@latest` I'm still being advised that I am on the previous version:

`npm list -g` shows I'm on the latest:

```bash
├── npm@9.6.7
├── opencommit@2.4.2
├── pm2@5.3.0
```

But `opencommit --version` returns the previous version, even when I run it from the direct path

```bash
╰─ /Users/timothybryant/.nvm/versions/node/v18.17.0/bin/opencommit --version
2.4.1
```

### Expected Behavior

After updating, `opencommit --version` should return the correct version. 

### Current Behavior

`opencommit --version` appears to return the wrong version. 

### Possible Solution

I feel like the tool is running the latest version, its just not reporting it when you run the `--version` flag. A few things I tried though were uninstalling the tool across all nvm versions and then reinstalling it to just one version. I also tried clearing my  npm cash with `npm cache clean -f`. 

### Steps to Reproduce

1) Install opencommit 2.4.1
2) Update to 2.4.2
3) Run `opencommit --version`

**Comment by di-sukharev** [comment]
thanks for the report, I will be publishing new version in the coming weekend, will fix this ;)

**Comment by ionutz89** [comment]
Any update about this issue?

**Comment by di-sukharev** [comment]
@ionutz89 not yet, for some reason when publishing a new package version, npm sets latest version correct, but the version which is being downloaded is `latest-1`, any volunteers to investigate this and make the world a better place?

**Comment by di-sukharev** [comment]
fixed

**Comment by robertoandrade** [comment]
Just tried upgrading to get this annoying warning to go and ended up breaking my local setup :\ reported as https://github.com/di-sukharev/opencommit/issues/247

**Comment by robertoandrade** [comment]
it's now fixed in 3.0.3
